lib/cli/src/commands/app/secrets/create.rs

```diff
@@ -0,0 +1,271 @@
+use super::utils::Secret;
+use crate::{
+    commands::{
+        app::util::{get_app_config_from_dir, prompt_app_ident, AppIdent, AppIdentFlag},
+        AsyncCliCommand,
+    },
+    opts::{ApiOpts, WasmerEnv},
+};
+use anyhow::Context;
+use colored::Colorize;
+use dialoguer::theme::ColorfulTheme;
+use is_terminal::IsTerminal;
+use std::{
+    collections::{HashMap, HashSet},
+    env::current_dir,
+    path::{Path, PathBuf},
+    str::FromStr,
+};
+use wasmer_api::WasmerClient;
+
+/// Create a new secret related to an Edge app.
+#[derive(clap::Parser, Debug)]
+pub struct CmdAppSecretsCreate {
+    /// The path to the directory where the config file for the application will be written to.
+    #[clap(long = "app-dir", conflicts_with = "app")]
+    pub app_dir_path: Option<PathBuf>,
+
+    /// The name of the secret to create.
```

Comment:
nit: I always like to put the positional args at the bottom so they can be identified easily.

Reply:
Reordered.